### DISCUSSION TITLE
How to create multiple swipers using SWIPER 6.4.1 on a single page

### DISCUSSION THREAD

**Discussion by James869-lab** [nolimits4web/swiper#4049] [Q&A]
This comes a very bad problem, how d o i create a multiple swipers on a single page using SWIPERS V.6.4.1? i've trie initiating class names,css names , and even javascript, the first swiper i imported on my website works but the second , looks like crap so can you help me guysss it;s realy urgent

**Comment by vltansky** [comment]
take a look here: https://github.com/nolimits4web/swiper/issues/273
